templates/expired.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+$this->data['header'] = 'Your password has expired';
```

Comment:
Translation support?  I've been building in French, Spanish and Korean on the material module.